### PR TITLE
config: add regression on checkstyle-samples ant

### PR DESCRIPTION
**Pull request by rnveach**
Identified at https://github.com/checkstyle/checkstyle/pull/12392#issuecomment-1348692764 .

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -7,6 +7,12 @@ language: java
 cache:
   directories:
     - ~/.m2
+    - ~/.ivy2
+
+addons:
+  apt:
+    packages:
+      - ant
```
Comment (romani) [inline .travis.yml:16]: We add and remove addons.
In some recent commit Nick suggested more stable way to install packages

Comment (rnveach) [reply .travis.yml:16]: I am not familar with any new ways. I just copied this from samples repo to reproduce it's CI.

Comment (romani) [reply .travis.yml:16]: Can you add ant to `sudo apt install -y .....`
That we already have for installation. Look at commit from my comment

Comment (rnveach) [reply .travis.yml]: I think done.

**Comment by rnveach** [comment]
> Other plugins executions 

What other do you refer to?

Also please set the next assignment or feel free to merge.

**Review by nrmancuso — APPROVED** [review]

**Pull request merged**